forms/collections/pages.js

```diff
@@ -35,12 +41,6 @@ define(function (require) {
       view.render();
       currentform.attributes._view.$el.append(view.el);
 
-      this.forEach(function (page, number) {
-        if (number !== index) {
-          page.removeView();
-        }
-      });
-
```

Comment:
its this loop that is causing double rendering, doesn't matter where in the function it is.

I think its fairly safe to just have the current page calling `removeView` - the `gotoPage` function controls the unloading and loading of a new page, If a dev is not using this function to display a page then they've likely customised things beyond what we should be trying to account for